README.md

```diff
@@ -18,16 +18,22 @@ To reach a shell within the Docker container, run
 docker compose run -it --remove-orphans web sh
 ```
 
+Migrate the database for the search app
+
+```sh
+bin/manage migrate search
```

Comment:
I love this simplification. If this is a commitment to maintaining the `bin` (which I think it should be) I have adjusted [our draft ADR](https://github.com/DOI-OS-ORDA/DARTS/pull/45) accordingly.

I think this step should clarify if it should be run in our out of the Docker container.